xarray/tests/test_dataarray.py

```diff
@@ -203,11 +203,11 @@ def test_sizes(self):
     def test_encoding(self):
         expected = {'foo': 'bar'}
         self.dv.encoding['foo'] = 'bar'
-        assert expected, self.d == encoding
+        assert expected == self.dv.encoding
```

Comment:
This change was odd.  I'm not sure what was expected here.

Reply:
Yeah, I think this change is correct. I have no idea what was going on previously -- `self.d` is not even a valid attribute.

Reply:
Previously, it asserted that `expected` was true-ish; if it wasn’t, it would print the value of `self.d == encoding` as the message, but that expression is only evaluated if the `assert` fails.

Reply:
I mean, that’s how the code was able to pass without error. Not sure what happened to cause that to be written. 😁